clar.c

```diff
@@ -211,7 +211,7 @@ static void clar_abort(const char *msg, ...)
 	va_start(argp, msg);
 	clar_print_onabortv(msg, argp);
 	va_end(argp);
-	exit(-1);
+	exit(1);
```

Comment:
I think that probably we should define exit codes then be consistent. I don't think that this is something that we need to address in this PR, just something for us to consider.

Reply:
Most of this has been superseded with the introduction of `clar_abort()` anyway, which does provide some consistency here. So there's only two cases left, and here we could think about specific error codes in a future MR indeed.